pkg/validator/app.go

```diff
@@ -172,12 +180,16 @@ func validateRoles(
 			},
 		)
 	}
-
-	if (appCR.Spec.DefaultSetupPackage.IsSet == false) || (appCR.Spec.DefaultSetupPackage.IsNull == true) {
+	if !appCR.Spec.DefaultSetupPackage.IsSet {
```

Comment:
Again nitpicking but can we reverse the blocks here? Like just have appCr.Spec.DefaultSetupPackage.IsSet() and then if it is not set initialize it to nil?

Reply:
Sticking to the pattern of the other blocks below I think, unless you'd prefer those be flipped too.

In any case lemme wait and see if there are any other review code changes that would require re-testing.